templates/site-logo-fragment.html

```diff
@@ -1,2 +1,6 @@
+{% if request.in_contest and request.participation.contest.logo_override %}
```

Comment:
Should also be shown on the `/contest/` page itself, even if you're outside of the contest.